### PR TITLE
Add date for completion of phase 2

### PR DESCRIPTION
**Pull request by AmberKiso**
## Summary
Add Nov. 15 as end of Phase 2.
## Testing Plan
Examined local web page
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: components/About/data.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export const phases = [
   },
   {
     status: Status.ComingSoon,
-    summary: 'Started May 10, 2022',
+    summary: 'Started May 10, 2022. Ending Nov. 15, 2022.',
```
Comment (AmberKiso) [inline components/About/data.tsx:202]: Not sure how I can force a newline in the framework, so truncated November instead.

File: pages/faq.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const questions: ReadonlyArray<{
   {
     question: `How long will the Phase 2 incentivized testnet run for?`,
     id: `testnet-duration`,
-    answer: `The testnet will run from May 10, 2022 @ 12:00 PDT until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
+    answer: `Phase 2 will run from May 10, 2022 @ 12:00 PDT until November 15, 2022. The testnet will run until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
```
Comment (AmberKiso) [inline pages/faq.tsx:26]: Wording is a little clunky. Could change to "The testnet will continue to run..."

**Comment by AmberKiso** [comment]
Overall, do we want to add "12:00pm PDT" or the equivalent in UTC (04:00???) wherever we mention an ending date?

File: components/Navbar/index.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ function Navbar({
 }: NavbarProps) {
   const notification = showNotification ? (
     <>
-      Phase 2 Incentivized Testnet starts May 10th @ 12:00 PDT!
+      Phase 2 Incentivized Testnet started May 10th @ 12:00 PDT and will end November 15th!
```
Comment (NullSoldier) [inline components/Navbar/index.tsx:30]: This is still hidden. We're gonna keep it hidden and were deploying the count down timer. We can show this after it ends.

```suggestion
      Phase 2 Incentivized Testnet ended November 15th @ 12:00 PDT
```

File: pages/faq.tsx (hunk 2 above)
Comment (NullSoldier) [reply pages/faq.tsx:26]: It's fine. Can we put the time as well? Noon PDT?

**Comment by NullSoldier** [comment]
I'm fine merging this, but do all of these changes make sense if we have a count down timer on every page? What do you  think?

**Comment by AmberKiso** [comment]
> I'm fine merging this, but do all of these changes make sense if we have a count down timer on every page? What do you think?

I think the clock handles notification of the end of Phase 2 well.
How about we use this commit for after the phase ends, since some of the "will end/ended" grammar would have to change anyway?

**Comment by NullSoldier** [comment]
Yes I'm fine with that.

File: pages/faq.tsx (hunk 2 above)
Comment (AmberKiso) [reply pages/faq.tsx]: Done

File: components/Navbar/index.tsx (hunk 3 above)
Comment (AmberKiso) [reply components/Navbar/index.tsx]: Done

**Comment by AmberKiso** [comment]
Changed ending date to Nov 22.
Do we want to mention anything about phase 3 in the FAQ/about pages? "Phase 3 planned to start January 2023"
I heard Jan 18th on the last monthly call.

**Comment by NullSoldier** [comment]
Look at the new staging branch and the phase 3 website, and see if you still want to make any more of these changes and reopen the PR.

**Pull request closed without merging**